src/Locator/ValidateLocator.php

```diff
@@ -37,6 +37,11 @@ protected function initFactories(array $factories)
             'between'               => function () { return new Validate\Between(); },
             'bool'                  => function () { return new Validate\Boolean(); },
             'callback'              => function () { return new Validate\Callback(); },
+            'caseLower'             => function () { return new Validate\CaseLower(); },
```

Comment:
I don't think validator probably need to check if it is lower / upper. Probably users can pass it to sanitize than in validation. But I don't have trouble though. Not sure if there is a right use case.